developerportal/templates/organisms/search-results.html

```diff
@@ -0,0 +1,30 @@
+<div id="results-list" class="search-results">
+{% if search_results %}
+  <h4 class="search-results-summary">
+    {{total_results}} result{{total_results|pluralize}} found for "{{search_query}}"
+  </h4>
+  {% if total_results > 1 %}
+  <p class="pagination-summary">
+    Showing results {{search_results.start_index}} to {{search_results.end_index}}
+  </p>
+  {% endif %}
```

Comment:
Without having interacted with the actual page/functionality, it feels like this can be an `article` element and the following two can wrapped as an `hgroup`. Something like:

```
<article id="results-list" class="search-results" aria-labelledby="results-heading">
{% if search_results %}
  <hgroup id="results-heading>
    <h4 class="search-results-summary">
      {{total_results}} result{{total_results|pluralize}} found for "{{search_query}}"
    </h4>
    {% if total_results > 1 %}
      <h5 class="pagination-summary">
        Showing results {{search_results.start_index}} to {{search_results.end_index}}
      </h5>
    {% endif %}
  </hgroup>
```

That way we create an explicit section and the heading exposed to assistive technologies would be something like:

```
10 results found for "iot" Showing results 1 to 5
```

if there is only 1 result it would be identified as:

```
1 result found for "iot"
```

This does also mean that `aria-labelledby="results-heading"` needs to wrapped in a conditional to ensure we only set the `hgroup` as the label for the section when there are actual results.

Reply:
Thanks! Change made and I'll stage it tomorrow AM